main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"nodewarmanager/config"
+)
+
+func main() {
+	err := config.Load()
+	if err != nil {
+		panic(err)
```

Comment:
It is fine to use `panic` here, but note that you should avoid using it in your internal libraries. It will crash the program with a stack dump. You can catch it with a `recover()` call, but it is better to deal with errors directly.
Read this article : https://eli.thegreenplace.net/2018/on-the-uses-and-misuses-of-panics-in-go/

Reply:
The idea is really to exit, but it seems that log.Fatal is a more graceful way of doing so!

Reply:
Also note that when the bot if fully developed, you will have some other tasks to do when gracefully stopping.